src/ui/popup/settings-page/index.tsx

```diff
@@ -1,22 +1,20 @@
 import {m} from 'malevic';
-import {isFirefox} from '../../../utils/platform';
 import {ViewProps} from '../types';
 import AutomationButton from './automation-button';
 import DevToolsGroup from './devtools';
-import EnabledByDefaultGroup from './enabled-by-default';
-import InvertPDF from './invert-pdf';
 import ManageSettingsButton from './mange-settings-button';
+import MiscellaneousButton from './miscellaneous-button';
 
 type SettingsPageProps = ViewProps & {
     onAutomationNavClick: () => void;
     onManageSettingsClick: () => void;
+    onMiscellaneousNavClick: () => void;
 };
 
 export default function SettingsPage(props: SettingsPageProps) {
     return (
         <section class="m-section">
-            <EnabledByDefaultGroup {...props} />
-            {isFirefox() ? null : <InvertPDF {...props} />}
+            <MiscellaneousButton onClick={props.onMiscellaneousNavClick}/>
```

Comment:
Maybe let's not add another page this time, but put the new option into "Manage Settings" page? Sounds like a suitable place for me.